src/cloud/utils/limits.ts

```diff
@@ -10,6 +11,11 @@ export const isLimitError = (error): boolean => {
 export const extractBucketLimits = (limits: LimitsState): LimitStatus => {
   return get(limits, 'buckets.limitStatus')
 }
+
+export const getBucketLimitStatus = (state: AppState): LimitStatus => {
```

Comment:
it's strange that this is scoped to app state, where as the other selectors are scoped to the subset of LimitState. I'm totally for using AppState (i view too large of a global namespace as a code smell), but you might want to reach out to watts to see why that jailing was made in the first place.

Reply:
Everywhere the limit selectors are used they are passed `state.cloud.limits` which I could not use with the `useSelector` hook

Reply:
i think you're missing the point here. everywhere you see those existing selectors, someone passed `state.cloud.limits` into it right? why?